resources/client_gen/swagger_templates/model.mustache

```diff
@@ -101,10 +101,10 @@ class {{classname}}(object):
             )
 {{/isListContainer}}
 {{#isMapContainer}}
-        if not set({{{name}}}.keys()).issubset(set(allowed_values)):
+        if not set({{{name}}}.values()).issubset(set(allowed_values)):
```

Comment:
Could this have an `or None` to satisfy the response from NiFi without breaking the codebase?